app/web/markdown/blog/2024/05/20/introducing-strong-verification.md

```diff
@@ -0,0 +1,51 @@
+
+Trust and Safety are a core tenet of usability for any couch surfing platform. At Couchers, we've spent a lot of time thinking about how we could make the platform safer and give people more control in order to use the platform according to their needs and preferences. Today we're announcing a new feature that gives users more power and peace of mind when using Couchers.org.
+
+**You can complete Strong Verification today, by vising the [new verification settings page](todo link).**
+
+## How does it work?
+
+To complete Strong Verification, you need your biometric passport, an NFC capable Android or iOS device, and the Iris ID app. You can download the app from the [Apple App Store](https://apps.apple.com/us/app/iris-id/id1575142357), or the [Google Play store](https://play.google.com/store/apps/details?id=seismic.rarity).
+
+You will first need to initiate Strong Verification in the new [verification settings page](todo link). You will be presented with a link or QR code to open in the Iris ID app. After following this link, you will be asked to scan the picture page of your passport, and then touch your NFC device to your passport while the information is downloaded from your passport. The data will be passed onto the Couchers server, encrypted, and you will receive a badge on your profile, as well as indicators for verified information.
+
+Your data will be securely stored on the server, with the most sensitive data encrypted and inaccessible to the server itself. You can read more about how we store and process your data [here](link).
+
+## Is Strong Verification free? Do I have to pay for it?
+
+**Every user will be able to complete Strong Verification for free until 1st October, 2024.**
+
+Strong Verification costs the Couchers, Inc. 501(c)3 non-profit real money. We do not currently have an abundance of funds, and so the board will have to make a decision regarding funding for this feature. Please [consider donating](https://couchers.org/donate).
+
+## What information is verified?
+
+We verify your date of birth and gender^[footnote: passports do not contain information about gender, only sex. We are forced to make a simplifying assumption regarding sex and gender. We also have a mechanism for those for whom this assumption does not hold, you can find out more [here](todo link).] match the details listed on your profile.
+
+## Why should I complete Strong Verification?
+
+By completing Strong Verification, other users will have the peace of mind that they are interacting with a real human. Strong Verification also stops people who were banned for violating the terms of service or community guidelines from creating new verified accounts^[footnote: Each passport can opnly be verified once, so users who are banned cannot re-verify using the same passport.]. Additionally, this data may be helpful if we ever need to collaborate with criminal investigations.
```

Comment:
Additionally, this data may be helpful if we ever need to collaborate with a criminal investigation.